src/compaction_filter.rs

```diff
@@ -116,22 +116,22 @@ fn test_filter(level: u32, key: &[u8], value: &[u8]) -> Decision {
 
 #[test]
 fn compaction_filter_test() {
-    use {Options, DB};
+    use {Options, TemporaryDBPath, DB, ops::*};
```

Comment:
Just out of curiosity. Why do we need `ops::*` here ?

Reply:
It's needed so all the database operations that are now implemented via traits are available. I think we should switch all the tests to use `prelude::*`, which includes all the `ops` traits. Rust 1.33 makes importing them a lot cleaner with [underscore imports](https://github.com/rust-lang/rfcs/blob/master/text/2166-impl-only-use.md), since the functions are available, but the traits themselves don't pollute the namespace.